include/swift/Frontend/FrontendOptions.h

```diff
@@ -199,7 +199,8 @@ class FrontendOptions {
   bool ImportUnderlyingModule = false;
 
   /// If set, the header provided in ImplicitObjCHeaderPath will be rewritten
-  /// by the Clang importer as part of semantic analysis.
+  /// by the Clang importer as part of semantic analysis if the ModuleOutputPath
+  /// for the primary input being compiled is not empty.
```

Comment:
That's not quite right; in practice this only happens when there *aren't* primaries. (That's an approximation of "are we emitting a swiftmodule for the entire module, rather than just one file?".) I would just leave the old comment, even though you're right that it doesn't happen if we don't actually output a swiftmodule.

Reply:
Removed what had been added to the comment.